support-workers/src/test/scala/com/gu/zuora/ZuoraITSpec.scala

```diff
@@ -0,0 +1,147 @@
+package com.gu.zuora
+
+import java.util.UUID
+
+import com.gu.config.Configuration.zuoraConfigProvider
+import com.gu.i18n.Currency.{AUD, EUR, GBP, USD}
+import com.gu.okhttp.RequestRunners
+import com.gu.support.workers.{GetSubscriptionWithCurrentRequestId, IdentityId, Monthly}
+import com.gu.support.zuora.api.response.{ZuoraAccountNumber, ZuoraErrorResponse}
+import com.gu.support.zuora.api.{PreviewSubscribeRequest, SubscribeRequest}
+import com.gu.test.tags.annotations.IntegrationTest
+import com.gu.zuora.Fixtures._
+import org.joda.time.{DateTime, DateTimeZone}
+import org.scalatest.flatspec.AsyncFlatSpec
+import org.scalatest.matchers.should.Matchers
+
+import scala.concurrent.duration._
+
+@IntegrationTest
+class ZuoraITSpec extends AsyncFlatSpec with Matchers {
+
+  def uatService: ZuoraService = new ZuoraService(zuoraConfigProvider.get(true), RequestRunners.configurableFutureRunner(30.seconds))
+
+  // actual sub "CreatedDate": "2017-12-07T15:47:21.000+00:00",
```

Comment:
I added this one line to confirm the created date of the subscription it is targeting